src/app/main/component/places/places.component.ts

```diff
@@ -153,10 +155,14 @@ export class PlacesComponent implements OnInit {
     }
   }
 
-  private bindLang(lang: string): void {
+  public bindLang(lang: string): void {
```

Comment:
We don't use this function in template - make it `private` instead of public

Reply:
Thank you! Changes are edded.